src/EditorFeatures/Core/AddImportOnPaste/AddImportOnPasteOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.AddImportOnPaste
+{
+    internal static class AddImportOnPasteOptions
+    {
+        /// <summary>
+        /// This option was previously "bool?" to accomodate different supported defaults
+        /// that were being provided via remote settings. The feature has stabalized and moved
+        /// to on by default, so the storage location was changed to
+        /// TextEditor.%LANGUAGE%.Specific.AddImportsOnPaste2 (note the 2 suffix).
+        /// </summary>
+        public static readonly PerLanguageOption2<bool> AddImportsOnPaste = new("dotnet_add_import_on_paste_options_add_imports_on_paste", defaultValue: true);
```

Comment:
Very meta question: should the names here be kept to be more "human readable" or do we really want the hierarchy like this? Because this could be simplified to just:

```suggestion
        public static readonly PerLanguageOption2<bool> AddImportsOnPaste = new("dotnet_add_imports_on_paste", defaultValue: true);
```

And that's a bit more human readable.

(separately, this is mixing "import" as being singular/plural which I could imagine could be a typo or not)

Reply:
> Very meta question: should the names here be kept to be more "human readable" or do we really want the hierarchy like this? Because this could be simplified to just:
> 
> And that's a bit more human readable.
> 
> (separately, this is mixing "import" as being singular/plural which I could imagine could be a typo or not)

Human readable like in editorconfig. Hierarchy should be achieved using `OptionGroup`, not the option name.

Reply:
I will rename all these option based on the suggestion here https://github.com/dotnet/roslyn/pull/66899#issuecomment-1434291638
And the grouping information we can move to OptionGroup